eng/devices/ios.cake

```diff
@@ -9,7 +10,7 @@ string TARGET = Argument("target", "Test");
 const string defaultVersion = "16.4";
 const string dotnetVersion = "net7.0";
 // required
-FilePath PROJECT = Argument("project", EnvironmentVariable("IOS_TEST_PROJECT") ?? "");
+FilePath PROJECT = Argument("project", EnvironmentVariable("IOS_TEST_PROJECT") ?? DEFAULT_APPIUM_PROJECT);
```

Comment:
We can't use the same thing since is Device tests and it uses a different path for device tests  we can only replace by DEFAULT_APPIUM_PROJECT in the ui-tests target

Reply:
Fixed. @ruiminhu - can you review the latest changes